test/tests/functional/pbs_cray_suspend_resume.py

```diff
@@ -96,6 +97,7 @@ def test_exclusive_job_not_suspended(self):
                               max_attempts=10)
         self.assertTrue(s)
 
+    @tags('cray', 'smoke')
```

Comment:
I don't believe this is a good smoke test (same reasoning as above).

Reply:
Done.